WordPress/Learn
# Add functionality for retrieving arrays of locale codes and labels

This sets the stage for us to have a locale picker in the UI for setting a workshop's language and captions, and subsequently filter workshops by these same values.

Much of the locale code was already in the wporg codebase, but it wasn't available in our local dev environment. So, a lot of the changes in this PR are related to making things available locally.

## To test

* You'll probably need to  rebuild the local dev environment:
  * `wp-env destroy`
  * `yarn run create`
* This doesn't currently surface the locale stuff anywhere in the UI or the front end of the site. Instead you can run some CLI stuff to see the output:
  * `wp-env run cli wp shell`
  * `\WordPressdotorg\Locales\get_locales_with_english_names()`
  * `\WordPressdotorg\Locales\get_locales_with_native_names()`
